examples/storage_system.rb

```diff
@@ -21,23 +21,50 @@
   password: ''
 }
 
-oneview_storage_system 'Teste' do
+# Example: add storage system or update if it already exists
+oneview_storage_system 'StorageSystem1' do
+  client my_client
   data ({
```

Comment:
I know it is coming from legacy code, but it would be good to have the `data ({  })` formatted like the others: `data( )`